<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statsdock</class>
 <widget class="QDockWidget" name="statsdock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stats</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::FieldsStayAtSizeHint</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="lblFps">
      <property name="text">
       <string>FPS:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="valueFps">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblDrawCount">
      <property name="text">
       <string>Draw Calls:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="valueDrawCalls">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblLuaCount">
      <property name="text">
       <string>Lua Object Count:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="valueLuaCount">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblLuaMemory">
      <property name="text">
       <string>Lua Memory:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="valueLuaMemory">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lblTextureMemory">
      <property name="text">
       <string>Texture Memory:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="valeuTextureMemory">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lblActionTree">
      <property name="text">
       <string>Action Tree time:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="valueActionTree">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lblNodeMgr">
      <property name="text">
       <string>Node Mgr time:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="valueNodeMgr">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="lblSimTime">
      <property name="text">
       <string>Sim time:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="valueSim">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="lblRenderTime">
      <property name="text">
       <string>Render time:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="valueRender">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
